studio/local_queue.py

```diff
@@ -74,6 +74,9 @@ def hold(self, key, minutes):
     def get_name(self):
         return 'local'
 
+    def shutdown(self, delete_queue=True):
+        self.delete()
```

Comment:
Should there be an if statement here ?

if delete_queue then
    self.delete()

Reply:
I don't think so. self.delete() here refers to the class instance itself,
so with delete_queue == False the whole shutdown() call will amount to nothing.
It's somewhat more involved for RabbitMQ, where there is our connection to RMQ server
which could be closed, and underlying RMQ queue which could be deleted OR left alone,
possibly to be connected to later.

Reply:
Got it, now I grok whats happening